pkgs/top-level/python-packages.nix

```diff
@@ -19123,6 +19123,29 @@ let
     };
   };
 
+  cmdtest = buildPythonPackage rec {
+    name = "cmdtest-${version}";
+    version = "0.15";
+    disabled = isPy3k || isPyPy;
+
+    buildInputs = with self; [ cliapp ttystatus ];
+    propagatedBuildInputs = with self; [ cliapp ttystatus markdown ];
+
+    # TODO: cmdtest tests must be run before the buildPhase
```

Comment:
Having looked into this further it can be fixed by patching yarn and cmdtest hashbangs to use /usr/bin/env python rather than /usr/bin/python. I will propose this upstream.